framework/utilities/ecl-u-disablescroll/ecl-u-disablescroll.twig

```diff
@@ -0,0 +1 @@
+<div class="ecl-u-disablescroll">...</div>
```

Comment:
[ ] the component supports composition
[ ] there are no hardcoded strings (all content come from the context)

The template can't be static like this, you could enable "children" elements to be passed inside. So that if the utility has a template, it's at least a wrapper to something which should not scroll.
Possible candidate in twig - [block](https://twig.symfony.com/doc/2.x/tags/extends.html#block-shortcuts).